<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>principal</class>
 <widget class="QMainWindow" name="principal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>principal</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(66, 239, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QWidget" name="widget" native="true">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(212, 251, 255);</string>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>311</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>registro de datos</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="btnCalcular">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>500</y>
         <width>80</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>calcular</string>
       </property>
      </widget>
      <widget class="QLabel" name="outTabla">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>210</y>
         <width>341</width>
         <height>191</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>350</y>
         <width>341</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(90, 255, 167);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>241</width>
         <height>88</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(175, 255, 204);</string>
              </property>
              <property name="text">
               <string>Ingrese la fecha</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateEdit" name="inFecha">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(175, 255, 204);</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(175, 255, 204);</string>
              </property>
              <property name="text">
               <string>ingrese su altura </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="inMetros">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(175, 255, 204);</string>
              </property>
              <property name="placeholderText">
               <string>altura en metros</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(175, 255, 204);</string>
            </property>
            <property name="text">
             <string>ingrese su peso</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="inPeso">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(175, 255, 204);</string>
            </property>
            <property name="placeholderText">
             <string>peso en kg</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QWidget" name="widget_2" native="true">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(212, 251, 255);</string>
      </property>
      <widget class="QPlainTextEdit" name="plainTextEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>371</width>
         <height>341</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>Aqui se mostra los datos </string>
       </property>
      </widget>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>500</y>
         <width>207</width>
         <height>24</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>IMC(kg/mÂ²):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="outImc">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>370</y>
         <width>271</width>
         <height>81</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Altura Maxima</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Altura Minima</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLineEdit" name="lineEdit"/>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_2">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionGuardar"/>
    <addaction name="actionCalcular"/>
    <addaction name="actionAbrir_ctr_a"/>
    <addaction name="separator"/>
    <addaction name="actionSalir_esc"/>
   </widget>
   <addaction name="menuArchivo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionGuardar">
   <property name="text">
    <string>&amp;Guardar </string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionCalcular">
   <property name="text">
    <string>&amp;Calcular </string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionAbrir_ctr_a">
   <property name="text">
    <string>&amp;Abrir </string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionSalir_esc">
   <property name="text">
    <string>&amp;Salir </string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
